app/models/user.rb

```diff
@@ -1,11 +1,19 @@
 class User < ActiveRecord::Base
   authenticates_with_sorcery!
+  before_create :create_organiser_token
 
   validates :password, length: { minimum: 6 }, if: -> { password || password_confirmation }
   validates :password, confirmation: true, if: -> { password || password_confirmation }
   validates :password_confirmation, presence: true, if: -> { password || password_confirmation }
 
-  validates :email, uniqueness: true
+  validates :email,  uniqueness: { case_sensitive: false } , presence: true,  format: { with: REGEXP_EMAIL }, on: :create
```

Comment:
There is no validation on email in edit form as email field is automatically taken as input and we dont ask from the user.